tests/integration/cli/projects/script/scripts/error_cli.py

```diff
@@ -1,7 +1,12 @@
 import click
 
+import ape
+
 
 @click.command(short_help="Use a subcommand")
 def cli():
     local_variable = "test foo bar"  # noqa[F841]
+    acct = "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045"
+    provider = ape.chain.provider
+    provider.set_balance(acct, 100000000)
```

Comment:
** nit ** can use more readable values i think, like "10 ETH", might require the chain method or wrapping in conversion convert